test/reviewsTest.js

```diff
@@ -81,31 +139,31 @@ describe('GET /reviews', () => {
   });
 
   it('returns correct reviews in correct format for valid product id 47421', async () => {
-    const response = await request.get('/product/47421');
+    const response = await request.get('/product/47421?sort=relevance');
     expect(response.body.product).to.eql('47421');
     expect(response.body.results.length).to.not.eql(0);
   });
 
   it('returns correct reviews in correct format for valid product id 47425', async () => {
-    const response = await request.get('/product/47425');
+    const response = await request.get('/product/47425?sort=helpfulness');
     expect(response.body.product).to.eql('47425');
     expect(response.body.results.length).to.not.eql(0);
   });
 
   it('returns correct reviews in correct format for valid product id 50001', async () => {
-    const response = await request.get('/product/50001');
+    const response = await request.get('/product/50001?sort=newest');
     expect(response.body.product).to.eql('50001');
     expect(response.body.results.length).to.not.eql(0);
   });
 
   it('returns correct reviews in correct format for valid product id 1', async () => {
-    const response = await request.get('/product/1');
+    const response = await request.get('/product/1?sort=relevance');
     expect(response.body.product).to.eql('1');
     expect(response.body.results.length).to.not.eql(0);
   });
 
   it('returns correct reviews in correct format for valid product id 100000', async () => {
-    const response = await request.get('/product/100000');
+    const response = await request.get('/product/100000?sort=helpfulness');
     expect(response.body.product).to.eql('100000');
     expect(response.body.results.length).to.not.eql(0);
   });
```

Comment:
clean looking tests! they are definitely testing your functionality to the T!!! nice work!